src/buildercore/cfngen.py

```diff
@@ -25,6 +27,27 @@
 
 LOG = logging.getLogger(__name__)
 
+FASTLY_AWS_REGION_SHIELDS = {
+    'us-east-1': 'iad-va-us', # N. Virginia: Washington
+    'us-east-2': 'iad-va-us', # Ohio: Washington
+    'us-west-1': 'sjc-ca-us', # N. California: San Jose
+    'us-west-2': 'sea-wa-us', # Oregon: Seattle
+    'ap-northeast-1': 'tokyo-jp2', # Tokyo: Tokyo
+    'ap-northeast-2': 'osaka-jp', # Seoul: Osaka
+    'ap-northeast-3': 'osaka-jp', # Osaka-Local: Osaka
+    'ap-south-1': 'singapore-sg', # Mumbai: Singapore (change to Mumbai when available)
+    'ap-southeast-1': 'singapore-sg', # Singapore: Singapore
+    'ap-southeast-2': 'sydney-au', # Sydney : Sydney
+    'ca-central-1': 'yul-montreal-ca', # Canada (Central): Montreal
+    'cn-north-1': 'hongkong-hk', # Beijing: Hong Kong
+    'cn-northwest-1': 'hongkong-hk', # Ningxia: Hong Kong
+    'eu-central-1': 'frankfurt-de', # Frankfurt: Frankfurt
+    'eu-west-1': 'london_city-uk', # Ireland: London City
+    'eu-west-2': 'london_city-uk', # London: London City
+    'eu-west-3': 'cdg-par-fr', # Paris: Paris
+    'sa-east-1': 'gru-br-sa', # São Paulo: São Paulo
```

Comment:
Going to open a Fastly issue since there doesn't seem to be a documented list anywhere (even of POP IDs).

Reply:
Opened 88013.

Reply:
Updated based on their list.